lib/Jsrt/Jsrt.cpp

```diff
@@ -1215,6 +1215,12 @@ CHAKRA_API JsStringToPointer(_In_ JsValueRef stringValue, _Outptr_result_buffer_
 
 CHAKRA_API JsConvertValueToString(_In_ JsValueRef value, _Out_ JsValueRef *result)
 {
+    if (value != nullptr && Js::JavascriptString::Is(value))
+    {
+        *result = value;
+        return JsNoError;
```

Comment:
Does this have any TTD impact? I suppose since this case explicitly doesn't have side effects, it should probably be fine.